src/library_tty.js

```diff
@@ -131,7 +145,8 @@ mergeInto(LibraryManager.library, {
           } else if (typeof window != 'undefined' &&
             typeof window.prompt == 'function') {
             // Browser.
-            result = window.prompt('Input: ');  // returns null on cancel
+            var result;
+            result = window.prompt('Input: ');   // returns null on cancel
```

Comment:
This could be just `var result = window.prompt('Input: ');`?

Reply:
Yeah, in fact there's already a 'var result  = null' at the top of the function, so this line isn't needed at all.